VERSIONING_STRATEGY.md

```diff
@@ -0,0 +1,67 @@
+# TSP Versioning Strategy
+
+At the moment, TSP is not yet at version 1.0.0.
+As such, we are expecting breaking changes to occur quite often until we reach V1.
```

Comment:
Even though it's not version 1.0.0, we try to not to make breaking changes often. Please rephrase that APIs will be added and updated until 1.0.0 as well as that there will be some breaking changes as well before.